logbook-core/src/main/java/org/zalando/logbook/CommonsLogFormatSink.java

```diff
@@ -0,0 +1,120 @@
+package org.zalando.logbook;
+
+import lombok.AllArgsConstructor;
+import org.apiguardian.api.API;
+
+import java.io.IOException;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.time.format.TextStyle;
+
+import static java.time.temporal.ChronoField.DAY_OF_MONTH;
+import static java.time.temporal.ChronoField.HOUR_OF_DAY;
+import static java.time.temporal.ChronoField.MINUTE_OF_HOUR;
+import static java.time.temporal.ChronoField.MONTH_OF_YEAR;
+import static java.time.temporal.ChronoField.SECOND_OF_MINUTE;
+import static java.time.temporal.ChronoField.YEAR;
+import static org.apiguardian.api.API.Status.EXPERIMENTAL;
+
+/**
+ * @see <a href="https://en.wikipedia.org/wiki/Common_Log_Format">Wikipedia: Common Log Format</a>
+ * @see <a href="https://httpd.apache.org/docs/trunk/logs.html#common>Apache HTTP Server: Common Log Format</a>
```

Comment:
missing closing `"` on the href

Reply:
:eagle: :eye:

Reply:
Fixed

Reply:
It killed the syntax highlighting of the class, therefore easy to spot for me :wink: